cli/admin/src/main/java/io/pravega/cli/admin/AdminCommand.java

```diff
@@ -472,6 +479,13 @@ private String objectToJSON(Object object) {
         return new ObjectMapper().writeValueAsString(object);
     }
 
+    public Controller instantiateController() {
+        ScheduledExecutorService executor = ExecutorServiceHelpers.newScheduledThreadPool(2, "Controller pool");
```

Comment:
Why do you need this? Other commands also make use of an executor without instantiating their own one. Also, why this logic is not placed in `ControllerCommand`? Looks more related to that.